src/styles/components/_NotFoundButton.scss

```diff
@@ -0,0 +1,48 @@
+.notfound-404 {
+    position: relative;
+    height: 200px;
+    h2 {
+      font-family: 'Montserrat', sans-serif;
```

Comment:
Good job @TheDrizzyWay. I think you may need to reconsider the font style here.  The font styles we have on our designs are `Raleway` and `Roboto`. You can use either of the two. It would be okay to stick to a max of two font styles.

Reply:
Roboto and Raleway it is then. Will fix this immediately